docs/content/en/docs/getting-started/assets/lifecycle-management.md_1.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnTaskDefinition
+metadata:
+  name: send-event
+  namespace: keptndemo
+spec:
+  retries: 0
+  timeout: 5s
+  container:
+    name: curlcontainer
+    image: curlimages/curl:latest
+    args: [
+        '-X',
+        'POST',
+        'http://webhook.webhook.svc.cluster.local:8084/YOUR-UUID-HERE',
+        '-H',
+        'Content-Type: application/json',
+        '-d',
+        '{ "from": "keptn send-event" }'
+    ]
```

Comment:
If possible, please increase the indentation

Reply:
Hi @odubajDT , what indentation is preferred 2 or 4 spaces?

Reply:
in this particular case 4, you can find all the details here https://github.com/keptn/lifecycle-toolkit/actions/runs/7111043259/job/19360404602?pr=2633